boa_engine/src/tagged.rs

```diff
@@ -0,0 +1,109 @@
+// Remove when/if https://github.com/rust-lang/rust/issues/95228 stabilizes.
+// Right now this allows us to use the stable polyfill from the `sptr` crate, which uses
+// the same names from the unstable functions of the `std::ptr` module.
+#![allow(unstable_name_collisions)]
+
+use sptr::Strict;
+use std::ptr::NonNull;
+
+/// A pointer that can be tagged with an `usize`.
+///
+/// Only pointers with an alignment of 2-bytes are valid, and the tag must have its MSB unset,
```

Comment:
Small doc clarity. Can MSB be listed out as `most significant byte (MSB)`. Same on line16 with `least significant byte (LSB)`. I'm actually not sure we need the acronyms in parenthesis, but it can't hurt.

Reply:
Solved and also reworded some other sentences.